microsoft-edge/dev/Dockerfile

```diff
@@ -50,7 +50,7 @@ RUN set -eux; \
 	} | tee /etc/default/microsoft-edge-dev
 
 # https://packages.microsoft.com/repos/edge/pool/main/m/microsoft-edge-dev/
-ENV EDGE_VERSION 120.0.2210.1-1
+ENV EDGE_VERSION 120.0.2186.2-1
```

Comment:
Something fishy is definitely going on -- maybe a weird/misbehaving cache?  When I fetch the packages list locally I get both 120.0.2210.1-1 _and_ 120.0.2210.4-1. :thinking:

Reply:
Ah, yep, definitely some misbehaving cache -- requesting it multiple times and I was able to reproduce. :upside_down_face: